win_configure/projects.VS2008/gen_pbs_version.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
+
+REM this genereates the pbs_version.h file.
+
+if exist "..\..\src\include\pbs_version.h.in" (
+COPY "..\..\src\include\pbs_version.h.in" "..\..\src\include\pbs_version.h" 
+)
```

Comment:
pbs_version.h.in is the file that configure will use as input to generate pbs_version.h. The idea is to use a staging machine to make a distributable source tar from the github sources, which will do the autotools step and create a complete tar source package for compilation on platforms that do not support autotools.

So, I dont think we should be copying pbs_version.h.in directly to pbs_version.h